src/core/tsi/ssl_transport_security.cc

```diff
@@ -1925,8 +1939,8 @@ tsi_result tsi_create_ssl_client_handshaker_factory_with_options(
   ssl_context = SSL_CTX_new(TLSv1_2_method());
 #endif
   if (ssl_context == nullptr) {
-    log_ssl_error_stack();
-    gpr_log(GPR_ERROR, "Could not create ssl context.");
+    gpr_log(GPR_ERROR, "Could not create ssl context: %s",
+            ssl_error_stack_to_string().c_str());
```

Comment:
Is the behavior changed for this line? IIUC, the original `log_ssl_error_stack` will clear the error queue. Will the queue be cleared if using the new `ssl_error_stack_to_string()` function?

Reply:
Ah, no.  The `ERR_clear_error()` call is redundant.  I'll remove that.

Reply:
Done, and updated method docstring.